build/FrameworkPackage/MakeFrameworkPackage.ps1

```diff
@@ -217,7 +226,16 @@ $compactxaml =
     </ResourceDictionary.MergedDictionaries>
 </ResourceDictionary>
 "@
-Set-Content -Value $compactxaml $fullOutputPath\Compact.xaml
+Set-Content -Value $compactxaml $DensityStylesDir\Compact.xaml
+
+# generate CustomizedPri
+$priConfigPath = [IO.Path]::GetFullPath("$CustomizedPriDir\priconfig.xml")
+$priOutputPath = [IO.Path]::GetFullPath("$CustomizedPriDir\CustomizedPri.pri")
+
+$makepriNew = "`"" + (Join-Path $WindowsSdkBinDir "makepri.exe") + "`" new /pr $CustomizedPriDir /cf $priConfigPath /of $priOutputPath /in $inputBaseFileName /o"
+Write-Host $makepriNew
+cmd /c $makepriNew
+if ($LastExitCode -ne 0) { Exit 1 }
```

Comment:
prefer using "& $makepriNew" instead of cmd /c. & is the [call operator]([https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_operators?view=powershell-6#call-operator](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_operators?view=powershell-6#call-operator)-).

Reply:
Also I see you copied this from below but maybe you can clean up both? 

---
In reply to: [277880000](https://github.com/Microsoft/microsoft-ui-xaml/pull/608#discussion_r277880000) [](ancestors = 277880000)